deployment_scripts/appimage/build_appimage.sh

```diff
@@ -0,0 +1,225 @@
+#!/bin/bash
+
+python_cmd="python3.7"
+APPDIR="$HOME/monkey-appdir"
+INSTALL_DIR="$APPDIR/usr/src"
+
+GIT=$HOME/git
+
+REPO_MONKEY_HOME=$GIT/monkey
+REPO_MONKEY_SRC=$REPO_MONKEY_HOME/monkey
+
+ISLAND_PATH="$INSTALL_DIR/monkey_island"
+MONGO_PATH="$ISLAND_PATH/bin/mongodb"
+ISLAND_BINARIES_PATH="$ISLAND_PATH/cc/binaries"
+
+is_root() {
+  return $(id -u)
+}
+
+has_sudo() {
+  # 0 true, 1 false
+  return $(sudo -nv > /dev/null 2>&1)
+}
+
+handle_error() {
+  echo "Fix the errors above and rerun the script"
+  exit 1
+}
+
+log_message() {
+  echo -e "\n\n"
+  echo -e "DEPLOYMENT SCRIPT: $1"
+}
+
+setup_appdir() {
+    rm -rf $APPDIR | true
+    mkdir -p $INSTALL_DIR
+}
+
+install_pip_37() {
+    pip_url=https://bootstrap.pypa.io/get-pip.py
+    curl $pip_url -o get-pip.py
+    ${python_cmd} get-pip.py
+    rm get-pip.py
+}
+
+install_nodejs() {
+  log_message "Installing nodejs"
+  node_src=https://deb.nodesource.com/setup_12.x
+  curl -sL $node_src | sudo -E bash -
+  sudo apt-get install -y nodejs
+}
+
+install_build_prereqs() {
```

Comment:
maybe add `sudo apt update` as well?

Reply:
The VM should be upgraded before building. I'll add details in a readme.

Reply:
Even if it's docker or something, how do you know that the repositories will be updated? I'd rather we have `sudo apt update` in scripts than specify in the readme that "you need to run sudo apt update".